src/hooks/useConnectRelatedVenues.ts

```diff
@@ -65,13 +70,19 @@ interface UseConnectRelatedVenuesReturn {
   venueEvents: WithVenueId<VenueEvent>[];
   siblingVenueEvents: WithVenueId<VenueEvent>[];
   subvenueEvents: WithVenueId<VenueEvent>[];
+
+  isRelatedVenuesLoaded: boolean;
+  isParentVenueLoaded: boolean;
+  isCurrentVenueLoaded: boolean;
 }
 
 export const useConnectRelatedVenues: ReactHook<
   UseConnectRelatedVenuesProps,
   UseConnectRelatedVenuesReturn
 > = ({ venueId, withEvents = false }) => {
-  const { currentVenue } = useConnectCurrentVenueNG(venueId);
+  const { currentVenue, isCurrentVenueLoaded } = useConnectCurrentVenueNG(
+    venueId!
+  );
```

Comment:
Why the `!` here?

```suggestion
  const { currentVenue, isCurrentVenueLoaded } = useConnectCurrentVenueNG(
    venueId
  );
```

Reply:
https://github.com/sparkletown/sparkle/pull/1047#discussion_r561820374